package/MDAnalysis/coordinates/CRD.py

```diff
@@ -182,23 +195,37 @@ def write(self, selection, frame=None):
             except AttributeError:
                 frame = 0  # should catch cases when we are analyzing a single PDB (?)
 
+        # account for explicit crd format, if requested
+        if self.crdext:
+            crdext = self.crdext
+        else:
+            crdext = crdext   # if not requested, default to NOEXT format
+
```

Comment:
I think this should all be removed and you use the class/init value in `self.extended`.
```suggestion
```